src/components/ebay-combobox/component.js

```diff
@@ -175,12 +183,13 @@ module.exports = {
                 this.el,
                 this.getEl('combobox'),
                 this.getEl('listbox'),
-                '[role="option"]', {
+                '[role="option"]',
+                {
                     activeDescendantClassName: 'combobox__option--active',
                     autoInit: -1,
                     autoReset: -1,
                     axis: 'y',
-                    autoScroll: true
+                    autoScroll: true,
```

Comment:
Is this a default prettier setting or something we enabled? Not a huge fan of these trailing commas. Looks so odd to me.

Reply:
Yeah it's a prettier setting. We can change it to 'trailingCommas: none' if we want to, but I think having trailing commas is the default. I feel like it's easier because then when you add an item to an array or object you don't have to go back to the previous one and add a comma

Reply:
Okay, despite my cognitive dissonance, let's keep the default setting to keep my aging brain on its toes (a brain with toes... weird imagery there).